lib/actions/ssh.coffee

```diff
@@ -98,11 +78,20 @@ module.exports =
 			.then ({ username, uuid, containerId, proxyUrl }) ->
 				throw new Error('Did not find running application container') if not containerId?
 				Promise.try ->
+					sshProxyCommand = ''
+					proxyConfig = global.PROXY_CONFIG
+					if proxyConfig
+						{ proxyAuth } = proxyConfig
+						proxyHost = "-p #{proxyConfig.host}:#{proxyConfig.port}"
+						proxyAuth = if proxyAuth then "-P #{proxyAuth}" else ''
```

Comment:
Oh, also just spotted a couple more points here:

* We should escape and quote `proxyAuth`. I think the other params are ok, but (assuming this is a username:password pair?) this could easily contain a single quote or a space, both of which would break this. Might be other cases that could cause trouble too, I'm not sure.

* Is `-P` right? That's not a listed argument in the proxytunnel docs: http://proxytunnel.sourceforge.net/usage.php